src/components/vmd-graph.base.component.scss

```
@@ -0,0 +1,14 @@
+@import "bootstrap/scss/functions";
+@import "bootstrap/scss/mixins/breakpoints";
+@import "../styles/bootstrap-variables";
+
+.chart-container-responsive {
+    position: relative; 
+    height: 250px;
+    width:72Vw;
+
+    @include media-breakpoint-up(md) {
+        height: 200px;
+        width:100%;
+    }
+}
```

```suggestion
canvas#graph {
	width: 100% !important;
	height: 200px;
}
```
J'ai essayé ainsi, ça a l'air de fonctionner, n'hésite pas à tester. L'idée, c'est simplement d'imposer la largeur de 100% (!important pour surchargé l'attribut style ajouté par Chart.js)
Et du coup à priori, la balise div.chart-container-responsive ne me semble plus forcément utile

N'hésite pas à me redire si tu vois des soucis avec cette façon de faire ! ;-)

---

@Aymerik Merci, je garde ton approche pour le canvas, c'est en effet plus simple et surtout ... ça marche !
Je garde juste la balise div.chart-container-responsive pour le comportement sur la hauteur du chart en mobile.